.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,66 @@
+name: IntegrationTests
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+    - 'Documentation/**'
+    - 'test/**'
+  push:
+    branches:
+    - master
+    - ft/master/**
+    paths-ignore:
+    - 'Documentation/**'
+    - 'test/**'
```

Comment:
Integration tests target runs files in this directory:

https://github.com/cilium/cilium/blob/cf73a56b3a4d21310dc8973f1876339502b653f9/Makefile#L174

Reply:
good catch :100: